opportunity/src/app/data/services/firebase.service.ts

```diff
@@ -12,11 +14,13 @@ import {map} from 'rxjs/operators'
 export class FirebaseCrudService {
   constructor(public db: AngularFirestore) {}
 
-  //gets
-  getOne(collection: string, objectKey: string) {
-    return this.db.collection(collection).doc(objectKey).valueChanges();
+  // getters
+  getOne(collection: string, id: string) { 
+    return this.db.collection(collection).doc(id).valueChanges();
```

Comment:
// .valueChanges() is simple. It just returns the 
    // JSON data without metadata. If you need the 
    // doc.id() in the value you must persist it your self
    // or use .snapshotChanges() instead. See the addItem()
    // method below for how to persist the id with
    // valueChanges()